README.md

```diff
@@ -1,51 +1,394 @@
+3. If the auto-detection of HLASM language does not recognize the file, set it manually in the bottom-right corner of the VS Code window.  
+4. The extension is now enabled on the open file. If you have macro definitions in separate files or use the COPY instruction, you must set up a workspace.
+
+### Setting Up a Multi-File Project Environment
+
+External files are usually accessed during HLASM evaluation (e.g. when the HLASM COPY instruction is used, or when macros are defined in external libraries). The source code interpreter in the HLASM Language Support extension must be set up correctly to be able to find the same files as the HLASM assembler program. 
+
+To do this, set up two configuration files — `proc_grps.json` and `pgm_conf.json`. Follow these steps:
+
+1. After you open a HLASM file for the first time, two pop-ups display. Select **Create empty proc_grps.json** and **Create pgm_conf.json with this file**. 
+   Two configuration files are then created with default values. They are stored in the `.hlasmplugin` subfolder.
+2. Navigate to the `proc_grps.json` file. This is the entry point where you can specify paths to macro definitions and COPY files. 
+3. Fill the `libs` array with the corresponding paths. For example, if you have your macro files in the `ASMMAC/` folder, add the string `"ASMMAC"` into the libs array.
 
+Follow the section [External Macro Libraries and COPY Members](#External-Macro-Libraries-and-COPY-Members) below for more detailed instructions on configuring the environment.
+
+The `pgm_conf.json` file can be provided implicitly by another product that supports integration with HLASM Language Support (e.g. Endevor Bridge for Git).
 
 ## Language Features
 
 The HLASM Language Support extension parses and analyzes all parts of a HLASM program. It resolves all ordinary symbols, variable symbols and checks the validity of most instructions. The extension supports conditional and unconditional branching and can define global and local variable symbols. It can also expand macros and COPY instructions.
 
 ## LSP Features
 ### Highlighting
-The HLASM Language Support extension highlights statements with different colors for labels, instructions, operands, remarks and variables. Statements containing instructions that can have operands are highlighted differently to statements that do not expect operands. Code that is skipped by branching AIF, AGO or conditional assembly is not colored.
+The HLASM Language Support extension highlights statements with different colors for labels, instructions, operands, remarks and variables. Statements containing instructions that can have operands are highlighted differently to statements that do not expect operands. Code that is skipped by branching `AIF`, `AGO` or conditional assembly is not colored.
 
-![](clients/vscode-hlasmplugin/readme_res/highligting.png)
+![](https://github.com/eclipse/che-che4z-lsp-for-hlasm/raw/master/clients/vscode-hlasmplugin/readme_res/highlighting.png)
```

Comment:
What's the reason for using absolute URLs here?

Reply:
Those files are included in the VSIX package. The issue with using absolute URLs is that you cannot examine the changes in PRs.

Reply:
Are you sure about this?
Absolute URLs for images in VS Code client readmes is what we have been doing since forever. We even create fake public repos for our non-OS extensions so that we can upload the images there. If we had the option of including them in the VSIX and linking to them with relative paths I am pretty sure we would be doing that.
Note that the images in the HLASM client readme have also always been called with absolute paths.

Reply:
That's exactly how it has been done so far and I am able to see all the resources in vscode just fine.

Reply:
Sorry but I really don't get what you mean, please go and look at the readme in the clients folder on the dev branch, the images are called with external paths. It's always been that way. (On the readme in the root folder, it doesn't matter, because that one doesn't display in the VS Code marketplace)

Reply:
ok, I see, this is something we will have to address in the pipeline. The packaging tool seems to have the ability to provide the base url for the resource referred to by the relative url.

So, for now, keep the top-level changelog with relative paths (this will be THE future readme) and I will look into the additional changes later on.